src/node_watchdog.h

```diff
@@ -46,11 +46,13 @@ class SigintWatchdog {
     void Dispose();
 
     v8::Isolate* isolate() { return isolate_; }
+    bool HasReceivedSignal() { return received_signal_; }
```

Comment:
Could be const.  It's considered idiomatic to name simple getters after their properties (i.e. `received_signal()`, although `has_received_signal()` is arguably a better name) but the current name is not wrong.